### ISSUE TITLE
Settle for an acronym to use in the codebase (PN, NM, ...)

### ISSUE THREAD

**Issue by toni-neurosc** [neuromodulation/py_neuromodulation#314]
Hi @timonmerk, I hope this discussion does not sound to dumb but in the recent refactoring I renamed the class Feature to NMFeature to make it less generic, NM standing for NeuroModulation. 
But now I realized that the base class for the Stream class is called PNStream, PN for PyNeuromodulation.
If we're going to keep on going with this naming practice, we should choose an acronym to use in all cases, whether it's NM, PN, PNM or PyNM (this one is too long maybe).

Or maybe we ditch the acronym altogether and just go for BaseFeature, or FeatureProtocol, BaseStream or sth like that.

**Comment by timonmerk** [comment]
Thanks a lot for raising this point! We should definitely keep the naming consistent throughout module names but also example imports, e.g. import py_neuromodulation as nm

We should also keep the acronym nm, that's across the features and preprocessing points established, so it would be good to keep it throughout the package.

**Comment by timonmerk** [comment]
I hope it's ok that I assigned you here @toni-neurosc, it could be part of #305 or even separate PR

**Comment by toni-neurosc** [comment]
Sure, np, I checked and there wasn't much to change.
- I change PNStream to NMStream 
- I change `import py_neuromodulation as pn` to `nm` in "test_multiprocessing.py"

In PR #305 all imports were changed to refer to the specific modules, and I think internally we should keep it this way so it's at all times clear at first glance what the inter-dependencies between modules are.

But when writing tests or demos/examples, then it should always read
```python
import py_neuromodulation as nm
```

So that would be about it for now, I guess we're all on the same page about this now? 

I'll maybe wait for  @SamedVossberg to confirm and then close it afterwards.
